deploy/packages/parallel-libraries.yaml

```diff
@@ -33,7 +33,7 @@ packages:
       - mpi
       - python
     specs:
-      - neuron@2018-10
+      - neuron
```

Comment:
Would it be so bad to list multiple versions explicitly?

Reply:
@pramodk if you're concerned about a lot of versions of neuron showing up in our modules, this needs to be fixed to a list of versions.